pkg/cli/decode.go

```diff
@@ -203,3 +214,74 @@ func convertFromUTF8(bytes []byte) (out []byte, ok bool) {
 	}
 	return out, true
 }
+
+//go:linkname gogoProtoFiles github.com/gogo/protobuf/proto.protoFiles
+var gogoProtoFiles map[string][]byte
+
+// dumpFileDescriptorSet outputs all registered file descriptors as a single FileDescriptorSet.
+func dumpFileDescriptorSet() error {
```

Comment:
style: IMO only the top level command function should access the static flag variables. helpers should take the flags as arguments.